app/pages/legal/PrivacyPolicy.jsx

```diff
@@ -6,7 +6,7 @@ export class PrivacyPolicyPage extends React.Component {
   static get content() {
     return `
 # Privacy Policy
-We do everything within our power to protect your privacy given the sensitive nature of many of the services listed on AskDarcel. 
```

Comment:
It seems like changes related to your other pull request ended up here -- best to keep them separate so we can review and merge them in one at a time.

Reply:
Ah totally that makes sense, thank you!